automation-tests/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Do we want to exclude automation tests from the mono repo build?  If not we should include that dir in workspaces list in the top level package.json so that the deps are installed/managed when the npm install is run. Also the package-lock.json file in the automation-tests folder should be deleted.

Reply:
Probably also want to update lerna.json so that we can run lint/format and such on that package along with the others.